lumicks/pylake/nb_widgets/image_editing.py

```diff
@@ -141,6 +165,13 @@ def update_plot(self, tether_coordinates):
             setter(lims)
         self.get_figure().canvas.draw()
 
+    def get_edited_image(self):
+        return (
+            self.current_image
```

Comment:
Does anyone still use `current_image`? Should it still be public? Otherwise it might be confusing because now there are two non-private images.